tests/modelwrapper_test.py

```diff
@@ -207,6 +216,15 @@ def test_predict_on_batch(self):
         pred = self.wrapper.predict_on_batch(input, 10, False)
         assert pred.size() == (2, 1, 10)
 
+        # iteration == 1
+        self.wrapper = ModelWrapper(self.model, self.criterion, replicate_in_memory=False)
+        pred = self.wrapper.predict_on_batch(input, 1, False)
+        assert pred.size() == (2, 1, 1)
+
+        # iterations > 1
+        pred = self.wrapper.predict_on_batch(input, 10, False)
+        assert pred.size() == (2, 1, 10)
+
```

Comment:
For the tests could we test that the outputs of iterations are different from each other?

Reply:
Yep, good idea :)

Reply:
I believe we are doing it [https://github.com/ElementAI/baal/blob/master/tests/modelwrapper_test.py#L260](here). The first Test is for multioutput model, since this has been tested in the case of `model_wrapper_test` we don't repeat it in multioutput.